### DISCUSSION TITLE
Dynamic routing is not working on Live Server

### DISCUSSION THREAD

**Discussion by akashswamy-maticz** [vercel/next.js#62570] [App Router]
Hi 
I have face dynamic routing issue in Live Server this my local dynamic Route http://localhost:3000/nft/detail/BNB/0x030f4eb42eefb52edf025da63daf78f877879367/1707894966/ 

my Live Server Router is  : https://phoenix-ai.maticz.in//nft/detail/BNB/0x030f4eb42eefb52edf025da63daf78f877879367/1707894966/

if i refresh the page its goes to 404 page 
 but its working fine in local 
 
my file structure 
![image_2024_02_27T07_25_27_355Zl](https://github.com/vercel/next.js/assets/110089002/301d913c-869b-420f-8411-0570c4b6dd85)

**Comment by icyJoseph** [comment]
Hi,

What is that page doing? Is it using getStaticProps, getServerSideProps?

**Reply by kameshmaticz** [reply]
we are using page router
we didt use any getStaticProps , getServerSideProps
we get data in ...details page like router.query.details

**Comment by akashswamy-maticz** [comment]
we are using page router 
we didt use any getStaticProps , getServerSideProps 
we get data in ...details page like router.query.details
